### DISCUSSION TITLE
Nest Hub (2nd Gen) Web GUI

### DISCUSSION THREAD

**Discussion by djandrew2005** [openthread/openthread#8601] [Q&A] [answered]
How can I access the web gui of the nest hub? It is an openthread border router so it must have one

**Comment by jwhui** [comment, marked as answer]
While Nest Hub (2nd Gen) implements Thread Border Router, it does not implement OpenThread's Border Router Web UI. The OTBR web UI is an optional component in the OTBR project and only meant for experimental use.

**Reply by djandrew2005** [reply]
Does it use Thread 1.3.0 or an older version? It requires a password in the Thread Commissioning App that I can't find anywhere

**Reply by jwhui** [reply]
> Does it use Thread 1.3.0 or an older version?

Nest Hub (2nd Gen) has Thread 1.3.0 Border Router Certification.

> It requires a password in the Thread Commissioning App that I can't find anywhere

Google Home App stores the Thread Credentials (a.k.a. Active Operational Dataset) using key stores provided by mobile OS platforms. On Android, you can use the [Thread Play Services APIs](https://developers.home.google.com/thread) to retrieve the Thread Credentials.
